packages/cli/src/commands/unlink.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+import unlink from '../scripts/unlink'
+import push from './push'
+
+const name = 'unlink'
+const signature = `${name} [libraries...]`
+const description = 'unlinks libraries from the project. Provide a list of whitespace-separated library names'
```

Comment:
maybe you have already thought about it, and it's not due to this PR but I noticed we are still using many terminologies for the same idea: stdlib, dependency, and library.. we should stick to only one or at least be clear with the difference between them
cc @spalladino

Reply:
Agree. It's a bit annoying and at this point the terminology has become a bit confusing. 
+1 If you want to discuss which one we should pick and why!